help/edit-a-message.md

```diff
@@ -0,0 +1,57 @@
+# Edit a message
+
+Zulip makes it possible to edit the content of your messages, letting you fix
+typos, clarify your thoughts, etc.
+
+Organization administrators can
+[configure](/help/restrict-message-editing-and-deletion) who can edit messages,
+and set time limits for this action. Administrators can never edit the content
+of a message.
```

Comment:
This is confusing, since they can edit their own messages.

"Even organization owners cannot edit the content of a message sent by another user."